code/modules/mob/living/brain/posibrain.dm

```diff
@@ -1,5 +1,50 @@
 GLOBAL_VAR(posibrain_notify_cooldown)
 
+GLOBAL_LIST_EMPTY(on_station_posis)
+
+/datum/job/cyborg/posibrain
+	title = JOB_NAME_POSIBRAIN
+	total_positions = 0
+	spawn_positions = 0
+	supervisors = "your laws" //No AI yet as you are just a cube
+	show_in_prefs = FALSE //No reason to show in preferences
+
+/datum/job/cyborg/posibrain/after_spawn_silicon(mob/living/silicon/robot/R, mob/M)
+	var/obj/item/mmi/posibrain/P = pick(GLOB.on_station_posis)
+
+	//Never show number of current posis
+	current_positions = 0
+
+	if(!P.activate(R)) //If we failed to activate a posi, kick them back to the lobby.
+		//Code taken from "send to lobby" admin panel
+		var/mob/dead/new_player/NP = new()
+		NP.ckey = M.ckey
+		qdel(M)
+
+		to_chat(NP, "<span class='warning'>Failed to Late Join as a Posibrain. Look higher in chat for the reason.</span>")
+
+	qdel(R)
```

Comment:
double qdel. actually `R == M` but stupid job code does that. line 22 is not necessary.